src/poggit/release/index/ReleaseListJsonModule.php

```diff
@@ -185,10 +185,18 @@ public function output() {
             }
         }
 
-        $isMin = substr(Meta::getModuleName(), -9) === ".min.json";
+        $isMin = str_ends_with(Meta::getModuleName(), ".min.json");
         header("Content-Type: application/json");
```

Comment:
See also https://github.com/wardi/jsonlines/issues/19

Reply:
Yeah, `application/json` clearly isn't correct here.
Since there does not appear to be any clear solution to that discussion should be use `application/x-ndjson` in the meantime?

Reply:
I guess nobody really cares about this header after all. Let's just keep it application/json in the meantime.